content/source/docs/enterprise/users-teams-organizations/organizations.html.md

```diff
@@ -4,14 +4,82 @@ page_title: "Organizations - Terraform Enterprise"
+
+Most of the organization settings are documented near the specific workflows they enable. What follows is a brief summary with links to more relevant sections of the documentation.
+
+### Profile / Delete
+
+The profile page shows the organization's name and contact email, but does not allow you to change them.
+
+The profile page is also where you can **delete your organization.**
+
+### Teams
+
+The teams page allows organization owners to manage the organization's teams, including creating and delete teams, managing membership, and generating team API tokens. See [Teams][] for more information.
+
+### VCS Provider
+
+The VCS provider page is used for setting up VCS access for TFE. See [Connecting VCS Providers](../vcs/index.html) for more information.
+
+### API Tokens
+
+Organizations can have a special service account API token that is not associated with a specific user or team. See [Service Accounts](./service-accounts.html) for more information.
+
+### Authentication
+
+The authentication page allows owners to determine when users must reauthenticate. It also allows owners to require [two-factor authentication](./2fa.html) for all members of the organization.
+
+### Manage SSH Keys
+
+The SSH keys page manages any keys necessary for cloning Git-based module sources during Terraform runs. It does not manage keys used for accessing a connected VCS provider. See [SSH Keys for Cloning Modules](../workspaces/ssh-keys.html) for more information.
+
+### Sentinel Policies
+
+Sentinel is an embedded policy-as-code framework that allows you to enforce rules about Terraform runs within an organization. See [Sentinel](../sentinel/index.html) for more information.
+
+## Inactive Organizations
+
+TFE is a paid product offered to organizations on a subscription basis; for more information about billing, please speak to a HashiCorp sales representative. TFE is also available as a free trial to organizations evaluating its features.
+
+When a free trial has expired, or when paid billing has been canceled, an organization becomes inactive. Inactive organizations display a banner reading "TRIAL EXPIRED — Upgrade Required" in the top navigation bar:
+
+![screenshot: the trial expiration banner](./images/org-inactive.png)
 
-If you navigate to TFE before you're a member of any organizations, you'll be asked to create one. You can also create a new organization at any time by going to an existing organization's settings, then clicking the "Create new organization" button in the sidebar navigation.
+Members of an inactive organization can still log into TFE, view data about past runs, manage team memberships, and manage access permissions on workspaces. However, inactive organizations cannot initiate new [Terraform runs](../run/index.html).
 
-Invite users to join an organization by sending them the
```

Comment:
I think we should leave a link here to the new home for this info, since org owners commonly wonder how to "invite" people.

Reply:
Got lost in the shuffle somehow, but yeah, definitely. I added an "adding users to orgs" header.